gc3apps/bf.uzh.ch/test/pymods/markovChain/mcInterface.py

```diff
@@ -92,16 +92,16 @@ def mcInterface(method, uncMean, persistence, varCov, nval, verb = 'DEBUG'):
     if not os.path.isfile(os.path.join(os.getcwd(), 'GHQUAD.DAT')):
       shutil.copy(os.path.join(pathToFile, 'GHQUAD.DAT'), os.getcwd())
     markovInputFile = open('PARMS', 'w')
-    print >> markovInputFile, len(nval)
-    print >> markovInputFile, 1
+    markovInputFile.write(len(nval) + "\n")
+    markovInputFile.write(str(1) + "\n")
     for ele in nval.flat:
-      print >> markovInputFile, ele
+      markovInputFile.write(ele + "\n")
```

Comment:
You should be able to do:

```python
print(whatever, file=markovInputFile)
```

here and in similar cases.

Reply:
In my first try, I inserted from __future__ import absolute_import where pylint suggested it, but this approach did not pass the tests. So, should I simply add it after the doc-string or before the first import?

I can certainly rebase - did you send a pull request? I didn't receive it.

Reply:
The `from __future__` line should be after the docstring and before the first import.

Pull request is here: https://github.com/malmstroem/gc3pie/pull/1#event-2232615404